sourcegraph/sourcegraph
# [SG-35853] Accessibility: Non-code file view: File action buttons don't respond to keypress

## Description
Migrated to `MenuLink` and `MenuItem` to support accessibility features such as keyboard navigation and press

## Problem description
![image](https://user-images.githubusercontent.com/105290007/177836191-0153dfce-6d4d-4a99-a19c-bf8ef4ec7888.png)
Pressing enter does not trigger these items (e.g. this one should link to GitHub, but instead dismisses the menu)

### Refs
[SourceGraph Issue](https://github.com/sourcegraph/sourcegraph/issues/35853)
[GitStart Issue](https://app.gitstart.com/clients/sourcegraph/tickets/SG-35853)

### Test Plan
- Run app on branch
- View a repository file on sourceraph e.g [here](https://k8s.sgdev.org/github.com/sourcegraph/sourcegraph@2a8a2da223d1e7655cac6eacbfc64f348d746ea5/-/blob/.gitignore)
- Confirm that you can trigger click on each dropdown item using your keyboard.

## App preview:

- [Web](https://sg-web-contractors-sg-35853.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-zwenjxpzrv.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
